src/transformers/models/vit_hybrid/modeling_vit_hybrid.py

```diff
@@ -166,11 +166,10 @@ def __init__(self, config, feature_size=None):
         feature_dim = self.backbone.channels[-1]
 
         if feature_size is None:
-            dummy_image = torch.zeros(1, num_channels, image_size[0], image_size[1])
-            with torch.no_grad():
-                feature_map = self.backbone(dummy_image).feature_maps[-1]
```

Comment:
You predicted correctly the failing test in https://github.com/huggingface/transformers/pull/20649#pullrequestreview-1208498147 @sgugger !